### PR TITLE
feat: add interface for error pages

### PR DESCRIPTION
**Pull request by saimonkat**
This PR brings interface for broken pages

**This is the continue for https://github.com/neondatabase/website/pull/2559, updated with fixes for header sidebar issue, which causes it's revert https://github.com/neondatabase/website/pull/2582**

Details:
- All pages have a classic full header
- GitHub stars are hidden from the [Error pages](https://neon-next-git-broken-pages-support-neondatabase.vercel.app/blog/wp-draft-post-preview-page?id=4410&status=draft) due to the client rendering

<details><summary>Before:</summary>

image
</details> 

After:
image

 TO-DO:
there is an issue for the [Draft Blog Post](https://neondatabase.wpengine.com/wp-admin/post.php?post=4410&action=edit) (made for the purpose of testing this PR), 
should be fixed after release (remove empty author)

[Preview](https://neon-next-git-broken-pages-support-neondatabase.vercel.app/blog/wp-draft-post-preview-page?id=4410&status=draft)

### CODE REVIEW

**Review by vannyle — APPROVED** [review]

**Comment by saimonkat** [comment]
@ruf-io we've updated this PR with fixes for missing header sidebar and tested all carefully
now all pages have classic header sidebar, except [Error](https://neon-next-git-broken-pages-support-neondatabase.vercel.app/blog/wp-draft-post-preview-page?id=4410&status=draft), where GitHub stars are hidden from  due to the client rendering

please take a look once again and approve if we good to merge 🙏

**Pull request merged**
